src/main/java/core/basesyntax/services/ReportMaker.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.services;
+
+public interface ReportMaker {
+    String createReport();
```

Comment:
Choose one of:
interface ReportMaker and makeReport() method
**OR**
interface ReportCreater and createReport() method